site/en/docs/extensions/mv3/architecture-overview/index.md

```diff
@@ -2,370 +2,107 @@
-the browsing experience without distracting from it. 
-
-The following is a list of the most common UI examples:
-
-- An [action click][docs-click] event.
-- A [popup][docs-popup].
-- A [context menu][docs-context-menu].
-- An [omnibox][docs-omnibox].
-- A [keyboard shortcut][docs-commands].
-- Desktop [notifications][api-notif].
-- [Text-to-speech][api-tts].
-- A custom UI injected [into a page][docs-content-scripts].
-
-See [Design the UI of a Chrome extension][docs-ui], to learn more.
-
-### Options page {: #optionsPage }
-
-Just as extensions allow users to customize the Chrome browser, the options page enables
-customization of the extension. Options can be used to enable features and allow users to choose
-what functionality is relevant to their needs.
-
-Users can access the options page via [direct link][docs-link-options] or in the context menu of the
-extension toolbar. The following is an example of the Google Dictionary extension. 
-
-{% Columns %}
-
-{% Column %}
-
-<figure>
-{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/Mz7GV76tFkzxRlb7Pq6e.png", 
-alt="Options page link in the UI", width="800", height="299" %}
-  <figcaption>
-    Link to the Options page.
-  </figcaption>
-</figure>
-
-{% endColumn %}
-
-{% Column %}
-
-<figure>
-{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/BM11QeGCThsUNTlsZbAe.png", 
-alt="Context Menu Options page", width="357", height="222" %}
+The manifest is the configuration file of a Chrome extension. It is a required JSON file that must be located at the root of the project. It provides the browser with a blueprint of the extension, with important information such as:
 
-  <figcaption>
-    Options page in the extension's context menu.
-  </figcaption>
-</figure>
+- The name of the extension, a description of what it does, the current version number, and what icons to use.
+- The [Chrome APIs][api-ref] keys and [permissions][doc-perms] that the extension needs.
+- The files assigned as the extension service worker, the popup HTML file, the options page, the content scripts, etc.
 
-{% endColumn %}
+The [Manifest keys][doc-manifest] article contains the complete list of default and optional keys. For copy-paste-ready code samples, check out the [Manifest examples][doc-manifest-examples].
 
-{% endColumns %}
+### The service worker {: #background_script }
 
-See [Give users options][docs-options] to learn more.
+The extension service worker is the event manager of a Chrome extension. It is a script that runs in the background within the context of the browser. It monitors and responds to events in the browser. For example, when a new tab is created, a new bookmark is added, the extension is installed, the extension toolbar icon is clicked, etc.
 
-### Additional HTML files {: #html-files}
+It can access all the [Chrome APIs][api-ref], but as a service worker, it can't use the DOM APIs that the global window object provides. It also cannot modify the page's content.
```

Comment:
> but as a service worker

I like that this line line is trying to draw a distinction between a service worker and an _extension_ service worker, but I don't think readers are likely to pick up on that here. I expect that people reading this document will see this as saying "the extension's service worker can access Chrome APIs, but as a service worker, it can't use DOM APIs".  I think that reads as oddly self-reflective and doesn't really address the question of why.

> It also cannot modify the page's content.

I find this line a bit confusing. As a reader, my immediate question is "_the_ page? What page is that?"

It's good to highlight that the background script cannot modify pages (something that has historically been a stumbling block for new extension developers), but I think we can lean a little more into this idea without substantially increasing the complexity of this paragraph.

**Suggestion**

```suggestion
A service workers can access all the [Extension APIs][api-ref], but as a type of [Worker](https://developer.mozilla.org/en-US/docs/Web/API/Worker) it can't use the typical DOM APIs that a document's global Window object provides. It also runs in its own environment, so it cannot directly modify a web page's content.
```